### DISCUSSION TITLE
Pull requests queue

### DISCUSSION THREAD

**Discussion by RomanArzumanyan** [NVIDIA/VideoProcessingFramework#530] [General]
Hi @maxclaey and @theHamsta !

Could you please take a look at PR queue? I understand you're busy with job duties you have.
Any chance I can help you with PR testing and merging? 
I'd like to submit more but I'm afraid to have my feature branch diverged from main repo.

**Comment by gedoensmax** [comment]
I am happy to merge basically anything you review. Full trust to your your judgement here. We currently do not have the time to manage VPF and we also lack the support due to reprioritization.

**Comment by RomanArzumanyan** [comment]
Thanks for the swift reply @gedoensmax 

> We currently do not have the time to manage VPF and we also lack the support due to reprioritization.

Any chance I can help you with this? 
Is it possible for me to take a maintainer role or company policies prohibits that?

**Comment by gedoensmax** [comment]
That's a very good question. As said I am happy to merge what you approve. I will also ask around how to handle this.

**Reply by RomanArzumanyan** [reply]
Hi @gedoensmax 

> I will also ask around how to handle this.

Any updates on this topic?

**Comment by RomanArzumanyan** [comment]
Hi @gedoensmax @theHamsta 

I totally understand that you are busy with your job duties and probably just don't have enough bandwidth to maintain this repository.
I appreciate the trust relationship and don't want to do anything behind your back.
So I'll clone this repo somewhere on GitHub and will continue working on it on my own.
If you're ok with it - please LMK, let's find out together how we can assist VPF users with migration. 
Otherwise if there are any concerns, alternatives or road blockers along this way - please LMK as well.
I'll wait till the end of October 2023 before forking off.

**Reply by gedoensmax** [reply]
@RomanArzumanyan thanks for your passion behind this repo and for putting the work ! I will ask around internally what how others feel about this. My personal feeling is positive though. I am very sorry for being so unresponsive here.
